### PR TITLE
Mobile: Fix and refactor PasswordFields

### LINKED ISSUE #1459: TypeError in src/ui/components/PasswordFields.js:94
## Error in Trinity Mobile

**TypeError** in **src/ui/components/PasswordFields.js:94**
undefined is not an object (evaluating 'this.state.feedback.warning')

[View on Bugsnag](https://app.bugsnag.com/iota-foundation/trinity-mobile/errors/5ca12f96486ce8001af3706c?event_id=5ccc82890038d4321be90000&i=gh&m=ci)

## Stacktrace

    src/ui/components/PasswordFields.js:94 - Component
    src/ui/views/onboarding/SetPassword.js:131 - Component
    src/ui/views/onboarding/SetPassword.js:201 - Component
    src/ui/components/DualFooterButtons.js:77 - PureComponent
    src/ui/components/DualFooterButtons.js:135 - PureComponent
    src/ui/components/Button.js:66 - PureComponent

[View full stacktrace](https://app.bugsnag.com/iota-foundation/trinity-mobile/errors/5ca12f96486ce8001af3706c?event_id=5ccc82890038d4321be90000&i=gh&m=ci)

*Created by Rajiv Shah via Bugsnag*

### PR DESCRIPTION
**Pull request by rajivshah3**
# Description

- Fixes crash when entering a password that is deemed insecure by `zxcvbn`
- Renames `this.state.score` to `this.state.zxcvbn` to reduce confusion with `this.state.score.score`
- Fixes type change by initializing `this.state.zxcvbn` as an object instead of a number

Fixes #1459 

## Type of change

- Bug fix (a non-breaking change which fixes an issue)
- Refactor

# How Has This Been Tested?

- Tested on iOS (debug)

# Checklist:

- [x] My code follows the style guidelines for this project
- [x] I have performed a self-review of my own code
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by rajivshah3** [comment]
Buildkite CI is failing because of a false positive report from `auditjs`

File: src/mobile/yarn.lock [hunk 1]
```diff
@@ -6001,10 +6001,10 @@ mkdirp@^0.5.0, mkdirp@^0.5.1:
   dependencies:
     minimist "0.0.8"
 
-moment-timezone@^0.5.21:
-  version "0.5.23"
-  resolved "https://registry.yarnpkg.com/moment-timezone/-/moment-timezone-0.5.23.tgz#7cbb00db2c14c71b19303cb47b0fb0a6d8651463"
-  integrity sha512-WHFH85DkCfiNMDX5D3X7hpNH3/PUhjTGcD0U1SgfBGZxJ3qUmJh5FdvaFjcClxOvB3rzdfj4oRffbI38jEnC1w==
+moment-timezone@^0.5.25:
```
Comment (laumair) [inline src/mobile/yarn.lock:6004]: @rajivshah3 Are these changes intended?

Comment (rajivshah3) [reply src/mobile/yarn.lock:6004]: I think `moment-timezone` was updated in #1506 but I forgot to update the lockfile in `mobile`

**Review by cvarley100 — APPROVED** [review]
